assertsprocessor/config.go

```diff
@@ -6,8 +6,9 @@ import (
 )
 
 type MatcherDto struct {
-	AttrName string `mapstructure:"attr_name"`
-	Regex    string `mapstructure:"regex"`
+	AttrName    string `mapstructure:"attr_name"`
+	Regex       string `mapstructure:"regex"`
+	Replacement string `mapstructure:"replacement"`
```

Comment:
We should provide a sensible default. I think it can be done as `mapstructure:"replacement", default:"$1"`